modules/imaging_browser/test/imaging_browserTest.php

```diff
@@ -160,13 +160,14 @@ public function setUp()
             )
         );
 
+        // @codingStandardsIgnoreStart
```

Comment:
I am afraid I can't.Phpcs complains about the long file name (value for key 'File') and I don't see a way to satisfy all of its requirements. Open to other suggestions...